modules/7_nlp/src/model.cpp

```diff
@@ -39,8 +46,37 @@ std::string SQuADModel::getAnswer(const std::string& question, const std::string
     tokens.push_back("[SEP]");
 
     std::vector<int> indices = tokenizer.tokensToIndices(tokens);
+    Blob::Ptr input = wrapVecToBlob(indices);
+    req.SetBlob("input.1", input);
+    req.Infer(); 
+    float* output1 = req.GetBlob(outputName)->buffer();
+    float* output2 = req.GetBlob("Squeeze_438")->buffer();
+    float max1 = output1[0], max2 = output2[0];
+    int indMax1 = 0, indMax2 = 0;
+    for (int i = 0; i < 128; i++) {
+        if (output1[i] > max1) {
+            max1 = output1[i];
+            indMax1 = i;
+        }
 
-    // TODO: forward indices through the network and return an answer
-
-    return "";
+        if (output2[i] > max2) {
+            max2 = output2[i];
+            indMax2 = i;
+        }
+    }
+    
+    std::string result;
+    for (int i = indMax1; i < indMax2 + 1; i++) {
+        std::string word = tokenizer.vocab[indices[i]];
+        if (word[0] == '#') {
+            result.pop_back();
+            result += word.substr(2, word.length());
```

Comment:
Please read the docs: http://www.cplusplus.com/reference/string/string/substr/. Second arguments should be length.